src/Dockerfile

```diff
@@ -1,7 +1,15 @@
 ARG CONTAINER="3.18"
 FROM alpine:${CONTAINER}
 # https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+
 ARG TARGETPLATFORM
+ARG WEB_BRANCH="development-v6"
+ARG CORE_BRANCH="development-v6"
+ARG FTL_BRANCH="development-v6"
+
+ENV TZ=Europe/Paris
```

Comment:
```suggestion
```

This actually does nothing in alpine it seems, however, we can add `tzdata` in a separate pull request, but leave it as default. Users can set the timezone at run time

https://stackoverflow.com/a/68996528